cni-plugin/integration/run.sh

```diff
@@ -152,8 +151,9 @@ flannel_overrides="{
                \"status\": {}
             }"
 # This needs to use the name linkerd-proxy so that linkerd-cni will run.
+# We use the calico overrides for cilium to avoid duplication.
 echo '# Running tester...'
-if [ "$SCENARIO" == "calico" ]; then
+if [ "$SCENARIO" == "calico" ] || [ "$SCENARIO" == "cilium" ]; then
```

Comment:
thought: Does `--overrides="$calico_overrides" \` have implications here? Just wondering if this should be another else if block rather than grouped with `calico`.

Reply:
I could break it into two if blocks but `cilium_overrides` would be 100% identical to `calico_overrides` and that bothered me.

I think what makes sense here is to rename `calico_overrides` to `generic_config_mount` and I've gone ahead and done that.